contributing/code/bc.rst

```diff
@@ -213,29 +213,29 @@ Change name                                         No
 Reduce visibility                                   No
 Move to parent class                                Yes
 Add argument without a default value                No
-Add argument with a default value                   No
+Add argument with a default value                   No [7]_ [8]_
 Remove argument                                     Yes [3]_
-Add default value to an argument                    No
+Add default value to an argument                    No [7]_ [8]_
 Remove default value of an argument                 No
 Add type hint to an argument                        No
-Remove type hint of an argument                     No
+Remove type hint of an argument                     No [7]_ [8]_
 Change argument type                                No
 Change return type                                  No
 **Protected Methods**
 Add protected method                                Yes
-Remove protected method                             No
-Change name                                         No
-Reduce visibility                                   No
+Remove protected method                             No [7]_
+Change name                                         No [7]_
+Reduce visibility                                   No [7]_
 Move to parent class                                Yes
-Add argument without a default value                No
-Add argument with a default value                   No
+Add argument without a default value                No [7]_
+Add argument with a default value                   No [7]_ [8]_
 Remove argument                                     Yes [3]_
-Add default value to an argument                    No
-Remove default value of an argument                 No
-Add type hint to an argument                        No
-Remove type hint of an argument                     No
-Change argument type                                No
-Change return type                                  No
+Add default value to an argument                    No [7]_ [8]_
+Remove default value of an argument                 No [7]_
+Add type hint to an argument                        No [7]_
```

Comment:
This can be allowed if the method is final because if you cannot extend the method, the added typehint (that is compatible of course) won't do harm.